tests/functional/api.php

```diff
@@ -0,0 +1,60 @@
+    ['?action=snippets&locale=it', 200, "{\n    \"@@this is a test, do not remove@@\": \"Questo \u00e8 un test\",\n    \"Email\": \"Email\",\n    \"Fifth string with %(one)s, %(two)s\": \"Quinta stringa con %(one)s, %(two)s e ancora %(two)s\",\n    \"Fourth string with 90%% coverage\": \"Quarta stringa con copertura al 90%\",\n    \"Hello\": \"Ciao\",\n    \"Obsolete string\": \"Stringa obsoleta\",\n    \"Press center\": \"Sala stampa\",\n    \"Save file\": \"Salva file troppo lungo\",\n    \"Second string with %(num)s tags\": \"Seconda stringa con %(num)s etichette\",\n    \"Sixth string with %(one)s\": \"Sesta stringa con %(one)s e %(two)s\",\n    \"String with %(num)s tags\": \"Stringa con etichette e variabile sbagliata\",\n    \"String with multiple comments\": \"String con pi\u00f9 commenti\",\n    \"Third string with %s tags\": \"Terza stringa con variabile mancante\"\n}"],
+];
+
+$obj = new \pchevrel\Verif('Check API HTTP responses');
+$obj
+    ->setHost('localhost:8083');
+
+$check = function ($object, $paths) {
+    foreach ($paths as $values) {
+        list($path, $http_code, $content) = $values;
+        $object
+            ->setPath($path)
+            ->fetchContent()
+            ->hasResponseCode($http_code)
+            ->isEqualTo($content);
+    }
```

Comment:
Since you removed the test on json feeds, you are not actually testing that we are sending valid json. You should separate your testing of pure text and json feeds or put in $paths what data you expect and add a conditional in the foreach loop to test for json or not.